docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -84,39 +91,49 @@ $ sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
 
 ## 4. Configure the compiler location
 
-Mbed CLI needs to know where the compiler is installed to build projects. It will look in these locations in the following order:
+To build project, Mbed CLI needs to know where the compiler is installed. It checks three configuration options in the following order:
 
-1. Mbed CLI config (local first, then global)
-1. Environment variables
-1. System path
+1. Mbed CLI config (local first, then global).
+1. Environment variables.
+1. System path.
+
+<!--the order in the list is the same as the original text, but the content has been changed so that PATH is first rather than last. Why?-->
 
 ### Install compilers in the system PATH
 
-Installing the compilers in the system PATH is most often the easiest method, as the compiler installation process will often place the executables in your system's PATH automatically.
+Installing the compilers in the system PATH is most often the easiest method, as the compiler installation process will often place the executables in your system's PATH automatically.<!--often? what does it depend on?-->
+
+<!--what was wrong with the original text
```

Comment:
I suppose it's personal preference. I typically don't care for sentences that start with "This means". I find having to explain a previous sentence (that isn't a directly quoting something) usually means the original sentence could be clearer.